pkg/project/cloud/aem/default/etc/aem.yml

```diff
@@ -139,15 +139,29 @@ instance:
         start_level: 20
         refresh_packages: true
 
-# Java options used to launch AEM instances
 java:
```

Comment:
@maciej-majchrzak-wttech how about such improvements and more explicit logic and configurable?